pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -477,8 +477,8 @@ const (
 	SecretForDockerRegistryV1GeneratorName = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName            = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName               = "configmap/v1"
-	ClusterRoleBindingV1GeneratorName      = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
-	RoleBindingV1GeneratorName             = "rolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
@CaoShuFeng we can't change the name of the generator (that's something users can specify on the CLI and we can't break them). We can add a second generator and change the default, but I'm not sure that's what we want either.

@kubernetes/sig-cli-pr-reviews how do we manage server compatibility crossed with generator versions?

Should we keep a single generator and have make different things based on what the target server supports, or have one generator per API version and select the default generator based on what the target server supports?